src/qml/MessageAlertBubble.qml

```diff
@@ -141,7 +141,7 @@ ListItemWithActions{
 
     Button {
         id: redownloadButton
-        text: i18n.tr("Download")
+        text: i18n.tr("Try again")
```

Comment:
Well, i think there is 2 cases here.
When "cellular data off", i think it would just need to have "Download" as a text
If "any error", the "Try again" is better yes.

Reply:
Is it that one button for both? I just see something about disabling it.
I meant to avoid
>tap the redownload button

not being called "redownload".